st2common/st2common/bootstrap/aliasesregistrar.py

```diff
@@ -132,6 +133,13 @@ def _register_action_alias(self, pack, action_alias):
         except StackStormDBObjectNotFoundError:
             LOG.debug('ActionAlias %s not found. Creating new one.', action_alias)
 
+        action_ref = action_alias_db.action_ref
+
+        action_db = Action.get_by_ref(action_ref)
```

Comment:
Let's see if we can speed this up.

I just made some changes in the past release which improve performance of similar code paths so instead of N queries (where N is number of aliases here) we only perform a single query.

I know this will likely require changing the code around though.

Reply:
@Kami I want to merge this PR and address bootstrap optimizations as part of another PR that I am working on. If you agree, I'll add tests for this one and try to get this merged?

Reply:
But please don't forget about it otherwise we will again make one step forward and one step back.